closure/goog/useragent/useragent.js

```diff
@@ -105,34 +94,34 @@ goog.userAgent.getNavigator = function() {
   // Need a local navigator reference instead of using the global one,
   // to avoid the rare case where they reference different objects.
   // (in a WorkerPool, for example).
-  return goog.global['navigator'] || null;
+  if ('navigator' in goog.global) {
```

Comment:
This returns `undefined` if `navigator` is set to `undefined`, potentially violating the type signature.  I don't know why that would ever be the case, but `|| null` is more likely to obey the signature, and it's not clear why this change makes anything better.